### PR TITLE
[backport][tests] KOGITO-4830: BPMN Read-only Mode E2E Coverage for standalone editors

### PR DESCRIPTION
**Pull request by domhanak**
Cherry-picks #466 to 0.9.0-prerelease.

### CODE REVIEW

**Review by jstastny-cz — APPROVED** [review]
LGTM.

File: packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+              cy.wrap($properties)
+                .find("[id='mainContainer']")
+                .should("be.visible")
+                .children(".row")
+                .should("have.length", 17)
+                .then($items => {
+                        propertyItems = $items;
+                });
+            });
+        });
```
Comment (tiagobento) [inline packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts:81]: It doesn't look like a good idea to have test cases depending on each other. Any way you can remove this `propertyItems` variable and make use of the `beforeEach`  method? Also, Is this properly formatted?

File: packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+              .should("not.be.disabled")
+              .should("be.visible")
+              .then($processVariablesContainer => {
+                    processVariables = $processVariablesContainer; 
+                    cy.wrap($processVariablesContainer)
+                      .find("#variableRow")
+                      .siblings()
+                      .should("have.length", "0");
+              });
+          });
```
Comment (tiagobento) [inline packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts:199]: Same here.

**Review by tiagobento — CHANGES_REQUESTED** [review]

**Comment by tiagobento** [comment]
Oh no! I just saw that this is targeting `0.9.0-prerelease`. This branch is not going anywhere after the 0.9.0 release 😁. Please consider sending a PR with the things I mentioned on the review to `master`.

**Comment by jstastny-cz** [comment]
> Oh no! I just saw that this is targeting `0.9.0-prerelease`. This branch is not going anywhere after the 0.9.0 release . Please consider sending a PR with the things I mentioned on the review to `master`.

Hello @tiagobento , this is an attempt to backport tests that are applicable to 0.9.0 version. They're already in master, but between 0.9.0 and master there were other changes that prevent us from running master tests against 0.9.0 release.

The ultimate goal is to have the tests versioned somewhere in a way that allows rerunning for a patch release or similar. What would be your recommendation? So far I've used my private fork, but that's not a good option.

**Comment by domhanak** [comment]
@tiagobento ok to merge

**Pull request merged**